tests/rotation_test.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (C) 2020-2021 - George Watkins and Alex Nguyen
```

Comment:
I recommend, adding the  `pauli_rotations` part to the path, like `tests/pauli_rotations/rotation_test.py`

Reply:
Since we went as far as adding a `src` folder I wonder whether it's wort adding the package name to the path:
```tests/lsqecc/pauli_rotations/rotation_test.py```

Reply:
No I don't think we need to do `tests/lsqecc` since we only have 1 package to test. We use `src` to make sure that other tools don't confuse between the test package and the actual package.

I am aligned with you on making the paths matched between tests and the package. We should also consider where to put our integration tests too.

Reply:
Integration tests usually go in a separate repo. We'd probably have them set up so you can choose to run them against `api.latticesurgery.com` or a local server.